packages/app/src/search/SearchEntry.tsx

```diff
@@ -0,0 +1,31 @@
+import { FiArrowRight } from 'react-icons/fi';
+
+import styles from '../explorer/Explorer.module.css';
+
+interface Props {
+  path: string;
+  isSelected: boolean;
+  onClick: () => void;
+}
+
+function SearchEntry(props: Props) {
+  const { path, isSelected, onClick } = props;
+
+  return (
+    <li key={path} className={styles.entity} role="none">
+      <button
+        className={styles.btn}
+        type="button"
+        role="treeitem"
+        data-path={path}
+        onClick={onClick}
+        aria-selected={isSelected}
+      >
```

Comment:
For now, since we show the full path, it'd be good to add a `title` attribute so users can see the full, uncropped path on hover.